api/scaler/dispatcher.go

```diff
@@ -0,0 +1,78 @@
+package scaler
+
+import (
+	"log"
+	"time"
+
+	"github.com/quintilesims/layer0/api/provider"
+)
+
+const (
+	SCALE_GRACE_PERIOD = time.Second * 15
+)
+
+var (
+	timeMultiplier time.Duration = 1
```

Comment:
What's the reason for doing
```
var (
		timeMultiplier time.Duration = 1
	)
```
instead of
```	
timeMultiplier := time.Duration(1)
```
? Not necessarily a change or a nit, just wondering what you know that I don't. :)

Reply:
I honestly didn't know you could do `timeMultiplier := time.Duration(1)` for package scoped variables. I thought it had to be `var x = y` or `var x type = y`